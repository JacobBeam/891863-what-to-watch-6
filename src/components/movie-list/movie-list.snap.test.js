import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import MovieList from './movie-list';

const films = [{id:0},{id:1}]
const countFilmsInList = 2
it(`Should MovieList render correctly`, () => {
  const history = createMemoryHistory();

  const { container } = render(
    <Router history={history}>
      <MovieList
        films={films}
        countFilmsInList={countFilmsInList}
      /></Router>
  )
  expect(container).toMatchSnapshot();
});
