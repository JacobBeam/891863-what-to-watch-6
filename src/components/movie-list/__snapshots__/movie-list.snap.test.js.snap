// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieList render correctly 1`] = `
<div>
  <div
    class="catalog__movies-list"
  >
    <article
      class="small-movie-card catalog__movies-card"
      data-film-id="0"
    >
      <video
        height="175"
        width="280"
      />
      <h3
        class="small-movie-card__title"
      >
        <a
          class="small-movie-card__link"
          href="/films/0"
        />
      </h3>
    </article>
    <article
      class="small-movie-card catalog__movies-card"
      data-film-id="1"
    >
      <video
        height="175"
        width="280"
      />
      <h3
        class="small-movie-card__title"
      >
        <a
          class="small-movie-card__link"
          href="/films/1"
        />
      </h3>
    </article>
  </div>
</div>
`;
